.github/workflows/generate-connector-registries.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         uses: actions/checkout@8edcb1bdb4e267140fa742c62e395cd74f332709 # v4.2.2
         with:
           ref: ${{ github.event.inputs.gitref || github.ref }}
+          submodules: true # Required for the enterprise repo since it uses a submodule that needs to exist for this workflow to run successfully.
```

Comment:
for my understanding: does this workflow run on the enterprise repo? I thought enterprise does a workflow call into the airbyte repo, and (I hope) this thing doesn't actually need anything from the airbyte repo

(... I haven't actually read the python code, though)

Reply:
from DM: this is just so that we can call the metadata_service python stuff